### DISCUSSION TITLE
Create a lib from our migrations helper

### DISCUSSION THREAD

**Discussion by anbraten** [woodpecker-ci/woodpecker#2148] [General]
It would be nice to use the migrations helpers from Woodpecker in other projects (exp Gitea) using xorm as well.

https://github.com/woodpecker-ci/woodpecker/blob/master/server/store/datastore/migration/common.go

**Comment by 6543** [comment]
https://github.com/go-gitea/gitea/blob/main/models/migrations/migrations.go#L522-L1016 is also worth a look

**Comment by qwerty287** [comment]
And the migration handling itself probably too.

**Comment by anbraten** [comment]
Got pointed to https://github.com/techknowlogick/xormigrate
